### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by rscohn2**

### CODE REVIEW

File: DPC++Compiler/vector-add/src/vector-add.cpp [hunk 1]
```diff
@@ -111,7 +111,7 @@ void VectorAddInDPCPP(const IntArray &addend_1, const IntArray &addend_2,
   // contains the data access permission and device computation (kernel)
   q.submit([&](handler &h) {
     // create an accessor for each buffer with access permission: read, write or
-    // read/write the accessor is the only mean to access the memory in the
+    // read/write. The accessor is the only means to access the memory in the
```
Comment (keryell) [inline DPC++Compiler/vector-add/src/vector-add.cpp]: Curious to have plural "means" here.

Comment (rscohn2) [reply DPC++Compiler/vector-add/src/vector-add.cpp]: [here is the definition](https://www.collinsdictionary.com/us/dictionary/english/means)

**Comment by JoeOster** [comment]
@rscohn2 - Thank you for your PR. We are now in the process of moving all oneAPI code samples to an Open Source repo on Github. Please look for the sample in this repo and make a new request when it becomes available.  

New GitHub Repo: https://github.com/oneapi-src/oneAPI-samples

Note: We will be closing this PR on 08/10/2020

**Comment by rscohn2** [comment]
Created a new PR in the new repo.

**Pull request closed without merging**